front-app/src/api/usersApi.tsx

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios';
+import { RegisterValidationInterface } from '../views/Register';
+import { LoginValidationInterface } from '../views/Login';
+
+const usersApi = axios.create({
+  baseURL: 'http://localhost:4000/users',
+});
+
+export const getUsers = async () => {
+  const response = await usersApi.get('/');
+  return response.data;
+};
+
+export const createUser = (user: RegisterValidationInterface) => usersApi.post('/register', user);
+
+export const login = (user: LoginValidationInterface) => usersApi.post('/login', user);
```

Comment:
Semantically, for me it doesn't make sense that a variable called user is of type `LoginValidationInterface`. Two option here, or changing the interface name or the variable name. It's a personal opinion buy maybe renaming user to data, or loginData makes more sense. This goes for createUser and login too